data-products/src/main/scala/org/sunbird/obsrv/dataproducts/model/MasterDataIndexerJobConfig.scala

```diff
@@ -0,0 +1,5 @@
+package org.sunbird.obsrv.dataproducts.model
+
+trait MasterDataIndexerJobConfig extends JobConfig {
```

Comment:
Please remove all these implementation and keep it very simple

Reply:
Ok